src/model/FeatureCol.js

```diff
@@ -10,9 +10,10 @@ const FeatureCol = Collection.extend({
     this.startOnCache = [];
     // invalidate cache
     this.on( "all", (function() {
+      this.assignRows();
```

Comment:
This assigns rows to features whenever the collection changes, effectively making it behave like a derived quantity.